public/app/features/alerting/unified/components/expressions/util.ts

```diff
@@ -37,10 +40,20 @@ const formatLabels = (labels: Labels): string => {
     .join(', ');
 };
 
+const formatRuleName = (namespace: CombinedRuleNamespace): string => {
```

Comment:
Shouldn't this method be named "formatFolderName" instead?

Reply:
Renamed it 👍